validations/src/validations.rs

```diff
@@ -197,7 +197,7 @@ pub fn validate_commit_collateral(
         let included_in_block_number = utxo_diff
             .included_in_block_number(input.output_pointer())
             .unwrap();
-        if included_in_block_number >= block_number_limit {
+        if included_in_block_number > block_number_limit {
```

Comment:
```suggestion
        // Outputs to be spent in commitment inputs need to be older than `block_number_limit`.
        // All outputs from the genesis block are fulfill this requirement because `block_number_limit` is always greater than `0`.
        if included_in_block_number > block_number_limit {
```

Reply:
Same concern here about whether my comment is true.